x/bank/types/params.go

```diff
@@ -51,28 +51,6 @@ func (se SendEnabled) Validate() error {
 	return sdk.ValidateDenom(se.Denom)
 }
 
-// validateSendEnabledParams is used by the x/params module to validate the params for the bank module.
-//
-//nolint:deadcode,unused
-func validateSendEnabledParams(i interface{}) error {
```

Comment:
The send enabled param is simply not supported anymore, I don't think we need to keep the validating logic (and it wasn't called either):

now
- https://github.com/cosmos/cosmos-sdk/blob/0943a70/x/bank/types/params.go#L34
in 0.46
https://github.com/cosmos/cosmos-sdk/blob/v0.46.4/x/bank/types/params.go#L49